app/views/posts/show.html.erb

```diff
@@ -1 +1,9 @@
 <h1>Specific post</h1>
+<div class='single-post'>
+  <h4>Post #<%= @post.id %> by <%= @post.author.name %></h4>
+  <span>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></span>
```

Comment:
- [ ] There is an error raised here, Please fix that.

![image](https://user-images.githubusercontent.com/68796232/223889148-697c5fd1-b5a1-463a-a5be-9ab2fd2be5e5.png)